docs/index.md

```diff
@@ -30,6 +30,16 @@ resource "grafana_folder" "my_folder" {
   org_id = grafana_organization.my_org.org_id
   title  = "Test Folder"
 }
+
+resource "grafana_dashboard" "test_folder" {
+  org_id = grafana_organization.my_org.org_id
+  folder = grafana_folder.my_folder.id
+  config_json = jsonencode({
```

Comment:
Is `jsonencode` the most common practice, or loading from disk? I would have assumed the latter, but don't actually know.

Reply:
It's pretty common practice. Any provider that requires JSON blobs uses it